src/main/java/frc/robot/subsystems/elevator/ElevatorInterface.java

```diff
@@ -2,9 +2,9 @@
 
 import org.littletonrobotics.junction.AutoLog;
 
-public interface ElevatorIO {
+public interface ElevatorInterface {
   @AutoLog
-  public static class ElevatorIOInputs {
+  public static class ElevatorInterfaceInputs {
```

Comment:
ElevatorInputs

Reply:
por favor

Reply:
plz correct this in all da files bc the elevator inputs are autologged from the specific impls. they aren't logged from the interface. @JacksonElia do u think this is good?

Reply:
yeah I think it is. I really think we should try to avoid using `default` and not implement any methods in interfaces as that defeats their purpose. If we really want to implement methods in the interface, we can just make them abstract classes.

Reply:
okay after looking into it more I guess its not awful using `default`. I looked at their template code and some things have it, but where they use it feels icky to me. I'm gonna ask eldest Matthew about it

Reply:
The reason I think we do that is bc for log replay u need a blank impl of the io layer to rerun the code. So we just pass in the interface. What we could do instead is create a blank impl with the defaults seen in the interface, which would work. Idk which one is better. So like as opposed to using `ElevatorInterface` we would use `ElevatorReplay` or smthn.

Reply:
Thats a good explanation, thanks. I would say normally I feel like making something like `ElevatorReplay` would be better practice, but in this case I guess following the convention of using default like akit has already specified makes more sense. 

Though, if we start writing a lot of replay specific code for whatever reason, then we should make a specific implementation for it.